optimizer.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+function optimizer (data) {
+    const objArray = Object.entries(data);
+    const modArray = objArray.map(a => a.map((b) => {

```

Comment:
This is a general solution for any objects with any keys. This is not our case and it's hard to understand. Especially with these variables' names.
You can specify values from entries as `key` and `value` and manipulate with them.

If this comment is unclear - write to me at slack

Reply:
I have added a new commit to the file with the relevant comment. I believe, you should be able to see it.
This thing now suggests me that it would be better to insert the code here. In case this is a better prictice, I will do so for the next comments.
Thank you